bugsnag-android-unity/src/main/java/com/bugsnag/android/unity/UnityClient.java

```diff
@@ -11,14 +11,27 @@
 public class UnityClient {
 
     static Client client;
+    static Configuration configuration;
```

Comment:
A `volatile` modifier will ensure that an up-to-date value of this variable can be seen on any thread.